.travis.yml

```diff
@@ -34,6 +34,27 @@ matrix:
     - env: TOX_SUFFIX="aiohttp"
       python: 3.7
       dist: xenial
+    - env: TOX_SUFFIX="lint"
+      python: 3.8-dev
+      dist: xenial
```

Comment:
A small cleanup suggestion: put `dist: xenial` on line 1 and delete all these.

Reply:
And see https://github.com/neozenith/vcrpy/pull/5 for some more suggestions.

Reply:
Awesome! Thanks for the review, the PR and the help with that. 

I'm kind of new to travis and tox and how they calculate the testing matrix but also how to specify one cell in that matrix too.

Any advice on where to read up about that or just comes with experience and tinkering?

Reply:
You're welcome!

It's generally experience and tinkering, you can have a look through these docs:
https://docs.travis-ci.com/user/build-matrix/
https://docs.travis-ci.com/user/customizing-the-build/#build-matrix

The 3.7/Xenial trick I knew from these alternatives https://github.com/travis-ci/travis-ci/issues/9815.